### PR TITLE
Document HTTP audit logger and token refresher

### PR DESCRIPTION
**Pull request by sebastianpoeplau**
Some things that I learned while using Friboo for the first time and that I think might be useful for others.

### CODE REVIEW

**Comment by dryewo** [comment]
👍

**Comment by hjacobs** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -190,6 +190,52 @@ Since the old audit log component does not suit our needs anymore, we introduce
...
+(audit/map->HTTP {:configuration {:api-url "https://audit.example.org/events"
+                                  :token-name :auditlog}})
+```
+
+Note that the HTTP audit logger needs access to the OAuth2 token refresher (see
+next section), configured with a token of the name specified above.
+
+### OAuth2 token refresher component
+
+The token refresher provides other componens with OAuth2 access tokens by means
```
Comment (shuaibiyy) [inline README.md:214]: typo: `componens`

Comment (sebastianpoeplau) [reply README.md:214]: Too late ;) Is there a way I can quickly fix the typo? I guess I'll have to go through a new merge request...

Comment (shuaibiyy) [reply README.md:214]: Didn't notice the PR had already been merged :D I think it's trivial enough to wait.

**Pull request merged**